github dep missing in package.json

Hi, this yields

```
module.js:340
    throw err;
    ^
Error: Cannot find module 'github'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/generator-gulp-plugin/app/index.js:20:17)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
```

when trying to tun `yo gulp-plugin` with version `0.3.0`.

--- a/package.json
+++ b/package.json
@@ -23,7 +23,8 @@
     "test": "mocha"
   },
   "dependencies": {
-    "yeoman-generator": "~0.14.0"
+    "yeoman-generator": "~0.14.0",
+    "github": "~0.1.12"
   },
   "devDependencies": {
     "mocha": "~1.14.0",

